backend/routes/setting_routes.py

```diff
@@ -0,0 +1,92 @@
+import uuid
+
+from flask import Blueprint, jsonify, request
+
+from db.models import Setting
+
+from resources.setting_resource import SettingResource
+from services import setting_service
+
+blueprint = Blueprint('setting', __name__, url_prefix='/api/settings')
+
+@blueprint.route('<int:id>', methods=['GET'])
+def get_setting(id):
+    """Gets setting by id
+
+    Returns:
+        setting with provided id
+    """
+    setting = setting_service.get_setting(id)
+    if setting is None:
+        error = {'error': 'Setting not found'}
+        return jsonify(error), 404
+
+    return jsonify(setting), 200
+
+@blueprint.route('/', methods=['GET'])
+def get_settings():
+    """Gets settings for the specified type
+
+    Request Body: 
+        setting_type: string
```

Comment:
Would be good to give some setting_type examples here or on Notion if it's not there yet

Reply:
Yup, I've got an enum for it setup in the DB.  See the new prisma.schema file on my branch.  Documentation will shortly follow on Notion.